src/main/java/seedu/address/model/Model.java

```diff
@@ -143,12 +143,22 @@
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
 
+    /**
+     * Wraps the filtered list of persons in a sortedList with a specific comparator
+     */
     void sortPerson(Comparator<Person> comparator);
+
+    /**
+     * Permanently sorts the address book by a specific comparator
+     */
+    void sortAddressBook(Comparator<Person> chooseComparator);
```

Comment:
I believe the name of the method would be more appropriate as something like `permSortContacts` or `permSortPersons` as we will eventually need to remove instances of 'addressbook' within the code. Additionally, I believe the argument name would be more appropriate as something like `comparator` as `chooseComparator` sounds more like a method name.